spec/integration_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper'
 require 'shinq'
 require 'shinq/client'
+require 'securerandom'
```

Comment:
This is because we use `SecureRandom.uuid` for test.
I assume some dependencies ceased to require `securerandom` at some point.